### PR TITLE
fix: change broken on-page gsoc 2023 links

### PR DESCRIPTION
**Pull request by bendehaan**

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ permalink: /initiatives/gsoc/gsoc2023ideas
 
 # {{page.title}}
 
-[ZAP](#owaspzap) | [Bug Logging Tool (BLT)](#bug-logging-tool-blt) | [Maryam](#owaspmaryam) | [SecureTea](#project-securetea) | [PyGoat](#project-pygoat) | [RiskAssessmentFramework](#risk-assessment) | [Juice Shop](#owaspjuiceshop)
+[ZAP](#owasp-zap) | [Bug Logging Tool (BLT)](#bug-logging-tool-blt) | [Maryam](#owasp-maryam) | [SecureTea](#owasp-securetea) | [PyGoat](#owasp-pygoat) | [RiskAssessmentFramework](#owasp-risk-assessment-framework) | [Juice Shop](#owasp-juice-shop) | [OWASP WrongSecrets](#owasp-wrongsecrets)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:10]: Amazingly we did actually check this earlier. It was working without hyphen (though maybe only on github not when deloyed :shrug:  ... with I could remember if we tried both :cry:)

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
Ya, this broke some when deployed. I'll fight with it later.

**Comment by kingthorin** [comment]
Some of them are rendering with `&nbsp;` for some reason, so the hyphens faile. Grr jekyll.

File: pages/initiatives/gsoc/gsoc2023ideas.md (hunk 1 above)
Comment (bendehaan) [reply pages/initiatives/gsoc/gsoc2023ideas.md:10]: I'm getting some inconsistent behaviour as well. Curious

**Comment by kingthorin** [comment]
Fixed

**Comment by bendehaan** [comment]
@kingthorin nice!

**Pull request merged**